### PR TITLE
PAYARA-3122 Managed Scheduled Executor Service still executes tasks after undeployment (Payara 4.x)

### PR DESCRIPTION
**Pull request by jGauravGupta**
Depends on : https://github.com/payara/Payara_PatchedProjects/pull/193

### CODE REVIEW

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 4 please

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 4 JDK7 please

File: appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ContextSetupProviderImpl.java [hunk 1]
```diff
@@ -301,9 +305,11 @@ private boolean isApplicationEnabled(String appId) {
                 // if app is null then it is likely that appId is still deploying
                 // and its enabled status has not been written to the domain.xml yet
                 // this can happen for example with a Startup EJB submitting something
-                // it its startup method. Reference Payara GitHub issue 204              
-                logger.info("Job submitted for " + appId + " likely during deployment. Continuing...");
-                result = true;
+                // it its startup method. Reference Payara GitHub issue 204
+                if(applicationRegistry.get(appId) != null){
```
Comment (arjantijms) [inline appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ContextSetupProviderImpl.java]:
```suggestion
                if (applicationRegistry.get(appId) != null) {
```

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 4 JDK7 please

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 4 please

**Pull request merged**